spec/support/sign_in_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SignInHelper
+  def sign_in
+    password = "shared_password"
+    create(:configuration, name: "admin:password",
+           value: BCrypt::Password.create(password))
+    visit root_path
+    fill_login_form(password: password)
+  end
+
+  def fill_login_form(password:)
+    fill_in "login", with: "test@example.com"
+    fill_in "password", with: password
+    click_on "Let me in!"
+  end
+end
+
+RSpec.configure do |c|
+  c.include SignInHelper, type: :feature
```

Comment:
I don't like the idea of splitting RSpec config into multiple files throughout `spec/support`, as opposed to just keeping it all in `spec_helper`.

Most other "spec modules" are being `include`d into RSpec in the same place, i.e. [in spec_helper]( https://github.com/dradis/dradis-ce/blob/master/spec/spec_helper.rb#L32). I think it would be better to include SignInHelper there, for consistency, rather than here.